### ISSUE TITLE
copyToClipboard not working on iOS 

### ISSUE THREAD

**Issue by diadal** [quasarframework/quasar#6378]
copyToClipboard not working on iOS no error log phone screen blink only when click on copy icon

```
import { copyToClipboard } from 'quasar'

copyToClipboard('some text')
  .then(() => {
    // success!
  })
  .catch(() => {
    // fail
  })
```
**Platform (please complete the following information):**
OS:
Node:
NPM:
Yarn:
Browsers:
iOS: iPhone XR simulator
Android:
Electron:

**Comment by rstoenescu** [comment]
What does the error in your catch() statement say?
If the iOS rejects the native call there's nothing that we can do. Does your app has enough perms?

**Comment by diadal** [comment]
where do I set permission for to allow this to work `catch()` only return `false` 

<img width="1005" alt="Screenshot 2020-02-15 at 15 00 15" src="https://user-images.githubusercontent.com/19895516/74589253-f98b4780-5003-11ea-8ff9-a78970d61df8.png">

**Comment by diadal** [comment]
this plugin work for me 

```
import VueClipboard from 'vue-clipboard2'

VueClipboard.config.autoSetContainer = true

```

**Comment by rstoenescu** [comment]
Enhancement will be available with "quasar" v1.9.3.
